stock_picking_partner_note/views/stock_picking.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model='ir.ui.view' id="view_picking_form_inherit">
+        <field name="name">Stock Picking Form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name="show_reserved" position="after">
+                <field name="partner_note_type_ids" widget="many2many_tags" />
```

Comment:
```suggestion
                <field name="note" />
```
I suppose we want to display the html here

Reply:
Do we want the customer to see the note on the report?
I thought is was a field for the WH employees.

Reply:
If the customer has access to the delivery documents then he can see the note.

Reply:
I just tested: 
On Picking Operations (for logistics)  we print the note 
On Delivery Note (for client) we don't print the note